assets/src/edit-story/components/library/text/fontPreview.js

```diff
@@ -70,13 +76,9 @@ function FontPreview({ title, element, onClick }) {
 
   return (
     <Preview onClick={onClick}>
-      <Text
-        fontSize={fontSize}
-        fontWeight={fontWeight}
-        fontFamily={font.family}
-      >
+      <PreviewText font={font} fontSize={fontSize}>
```

Comment:
Should this still be `fontFamily={font.family}`?

Reply:
I was wondering this too! It looks like there's no font family on the text presets here: https://github.com/google/web-stories-wp/blob/main/assets/src/edit-story/components/library/panes/text/textPresets.js

happy to add it back in though!